### PR TITLE
Remove unused dart:async imports

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**
Since Dart 2.1, Future and Stream have been exported from dart:core

### CODE REVIEW

**Comment by mk13** [comment]
Hmm, is the removal of async causing the strange `InconsistentAnalysisException: Requested result might be inconsistent with previously returned results` issue in Travis?

**Comment by MichaelRFairhurst** [comment]
I'm not sure what's going on.

This package doesn't depend on Dart 2.11, and the 2.11 SDK is compatible with a 2.10 analyzer.

I'll try bumping the analyzer version, though.

**Comment by mk13** [comment]
Seems like there's a `pub get` issue when used with the `analyzer` versioning?

**Comment by dotdotcommadot** [comment]
@mk13 What fixed the `InconsistentAnalysisException` issue for me, was that the some of the builders (like a SASS builder in build.yaml) should have a property `runs_before: [pageloader]`. Don't know if it's of any help for you though.

File: pubspec.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
 
 dependencies:
-  analyzer: '>=0.36.0 <0.40.0'
+  analyzer: '>=0.36.0 <0.41'
```
Comment (whesse) [inline pubspec.yaml:20]: This needs to be 0.41.0.
I suspect that the failures on Travis are not caused by the async imports removal, but due to the SDK Travis uses for testing being updated.  So an empty PR that just fixes the Travis errors would unblock all the pending PRs that are failing on Travis.

**Comment by mk13** [comment]
Handled with #217.

**Pull request closed without merging**